src/goto-analyzer/static_verifier.cpp

```diff
@@ -262,6 +262,31 @@ static void static_verifier_console(
     m.result() << '\n';
 }
 
+static static_verifier_resultt::statust
+check_assertion(const ai_domain_baset &domain, exprt e, const namespacet &ns)
+{
+  if(domain.is_bottom())
+  {
+    return static_verifier_resultt::BOTTOM;
+  }
+
+  domain.ai_simplify(e, ns);
+  if(e.is_true())
+  {
+    return static_verifier_resultt::TRUE;
+  }
+  else if(e.is_false())
+  {
+    return static_verifier_resultt::FALSE;
+  }
+  else
+  {
+    return static_verifier_resultt::UNKNOWN;
+  }
+
+  UNREACHABLE;
```

Comment:
I believe the `UNREACHABLE` is unnecessary as this place is just syntactically unreachable right now

Reply:
It is right now, I tend to use them defensively so I can't later break the code by removing an option / adding a value to the enum without finding out.